modules/dnn/src/op_webnn.hpp

```diff
@@ -31,6 +31,66 @@ constexpr bool haveWebNN() {
 #endif
 }
 
+#ifdef HAVE_WEBNN
+
+class WebNNBackendNode;
+
+
+class WebNNGraph
+{
+public:
+    WebNNGraph();
+
+    bool isInitialized();
+
+    void init(Target targetId);
+
+    void forward(const std::vector<Ptr<BackendWrapper> >& outBlobsWrappers);
+
+    void createGraph(Target targetId);
+
+private:
+    ml::GraphBuilder builder;
+    ml::Context mContext;
+    ml::Graph mGraph;
+    ml::NamedResults mResults;
+    bool isInit = false;
+};
+
+class WebNNBackendNode : public BackendNode
+{
+public:
+    WebNNBackendNode(const std::vector<Ptr<BackendNode> >& nodes, Ptr<Layer>& layer,
+                        std::vector<Mat*>& inputs, std::vector<Mat>& outputs,
+                        std::vector<Mat>& internals);
+
+    void setName(const std::string& name);
+
+    std::shared_ptr<ml::Operand> operand;
+};
+
+class WebNNBackendWrapper : public BackendWrapper
+{
+public:
+    WebNNBackendWrapper(int targetId, const Mat& m);
+    WebNNBackendWrapper(Ptr<BackendWrapper> wrapper);
+    // ~WebNNBackendWrapper();
+
+    static Ptr<BackendWrapper> create(Ptr<BackendWrapper> wrapper);
+
+    virtual void copyToHost() CV_OVERRIDE;
+    virtual void setHostDirty() CV_OVERRIDE;
+    virtual void * getBuffer();
+
+private:
+    void * buffer;
+};
+
+#endif  // HAVE_WebNN
+
+void forwardWebNN(const std::vector<Ptr<BackendWrapper> >& outBlobsWrappers,
+                   Ptr<BackendNode>& operand);
```

Comment:
Keep indent aligned to the first parameter in the previous line.

Reply:
Fixed.